confs/pump.env

```diff
@@ -0,0 +1,30 @@
+TYK_PMP_OMITCONFIGFILE=true
```

Comment:
Is this a real env variable? I can't find it anywhere

https://github.com/TykTechnologies/tyk-pump/blob/master/config.go

Reply:
[Found it](https://github.com/TykTechnologies/tyk-pump/commit/73d4d3f88740), it's not released yet.  Let's keep it in.